core/src/main/scala/org/apache/spark/storage/FallbackStorage.scala

```diff
@@ -119,6 +120,27 @@ object FallbackStorage extends Logging {
     }
   }
 
+  /** Clean up the generated fallback location for this app. */
+  def cleanUp(conf: SparkConf, hadoopConf: Configuration): Unit = {
+    if (conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH).isDefined &&
+        conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_CLEANUP) &&
+        conf.contains("spark.app.id")) {
+      val fallbackPath =
+        new Path(conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH).get + "/" + conf.getAppId)
```

Comment:
nit:

```suggestion
        new Path(conf.get(STORAGE_DECOMMISSION_FALLBACK_STORAGE_PATH).get, conf.getAppId)
```

Reply:
(BTW, please just don't bother about the co-author stuff manually @dongjoon-hyun. I think it's fine to follow GitHub Actions' policy for now)

Reply:
Thank you, @HyukjinKwon . Got it. I'll merge here via GitHub.